go.mod

```diff
@@ -5,8 +5,9 @@ go 1.17
 require (
 	github.com/SkynetLabs/go-skynet/v2 v2.0.2
 	github.com/dgraph-io/badger/v3 v3.2103.1
-	github.com/docker/distribution v2.7.1+incompatible
```

Comment:
I wonder where are we using distribution?

Reply:
@guacamole let me do it via another PR